src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.netcoreapp.cs

```diff
@@ -417,16 +417,6 @@ public void UnixDomainSocketEndPoint_UsingAbstractSocketAddressOnUnsupported_Thr
             }
         }
 
-        [ConditionalFact(nameof(PlatformDoesntSupportUnixDomainSockets))]
-        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Why don't we want to keep this eg for windows 7?

Reply:
It was cumbersome to keep it and deal with the subtleties of when it was supported.  Just didn't seem worth it.  I could easily keep it if all we care about is testing <= RS3.